test/helpers/cliCommandRunner.js

```diff
@@ -0,0 +1,128 @@
+// Import Node.js Dependencies
+import { fork } from "node:child_process";
+import { createInterface } from "node:readline";
+
+// Import Third-party Dependencies
+import { MockAgent, setGlobalDispatcher } from "undici";
+
+// Import Internal Dependencies
+import { getCurrentRepository } from "../../src/commands/scorecard.js";
+
+function initChildProcess({ path, args, mockApiOptions }) {
+  const childProcess = fork(path, args, {
+    stdio: ["ignore", "pipe", "pipe", "ipc"]
+  });
+  childProcess.send(mockApiOptions);
+
+  return childProcess;
+}
+
+async function forkAndGetLines(options) {
+  const childProcess = initChildProcess(options);
```

Comment:
Feel like initChildProcess function is not required here (would prefer to move the code from it directly here).